PyISY/Nodes/__init__.py

```diff
@@ -187,8 +187,22 @@ def parse(self, xml):
                                 dimmable = '%' in \
                                     nprop[0].attributes['uom'].value
                                 nval = int(nval.replace(' ', '0'))
+                            # Get the device notes, currently only the Spoken property is saved.
+                            notes_xml = self.parent.conn.getNodeNotes(nid)
```

Comment:
Understood, my application needs to know all the spoken properties always, so it made sense.  But, if you never need them, it does add some time to startup.

So are you saying if the caller asks for the spoken property and it hasn't been checked, then make the rest call?